code/modules/clothing/clothing.dm

```diff
@@ -330,6 +332,26 @@ BLIND     // can't see anything
 		"Vox" = 'icons/mob/species/vox/shoes.dmi'
 		)
 
+/obj/item/clothing/shoes/attackby(obj/item/weapon/W, mob/user, params)
+	if(istype(W, /obj/item/weapon/wirecutters))
+		if(can_cut_open == 1)
+			if(cut_open == 0)
+				playsound(src.loc, 'sound/items/Wirecutter.ogg', 100, 1)
+				user.visible_message("\red [user] cuts open the toes of [src].","\red You cut open the toes of [src].")
+				cut_open = 1
+				if("exclude" in species_restricted)
+					name = "mangled [name]"
+					desc = "[desc] They have had their toes opened up."
+					species_restricted -= "Unathi"
+					species_restricted -= "Tajaran"
+					icon_state = "[icon_state]_opentoe"
+					item_state = "[item_state]_opentoe"
+			else if(cut_open == 1)

```

Comment:
No idea why I put the icon updates down there, they should be just below cut_open=1 and above the if("exclude"...